src/index.html

```diff
@@ -22,6 +22,391 @@
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h2 class="card__review-number">Reviews: 5</h2>
+          </div>
+          <div class="card__price">
+            <p class="card__price-name">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </button>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac27"
+          />
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h1>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h2 class="card__review-number">Reviews: 5</h2>
+          </div>
+          <div class="card__price">
+            <p class="card__price-name">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </button>
+        </div>
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac27"
+          />
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </h1>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <h2 class="card__review-number">Reviews: 5</h2>
+          </div>
+          <div class="card__price">
+            <p class="card__price-name">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </button>
+        </div>
```

Comment:
The product cards are repeated multiple times with the same content. Consider dynamically generating these cards using JavaScript to avoid redundancy and improve maintainability. This aligns with the previous review's suggestion to ensure product cards are dynamically generated if repetition was not intended.